homeassistant/components/solaredge/coordinator.py

```diff
@@ -94,8 +94,10 @@ def update(self) -> None:
             for index, key in enumerate(energy_keys, start=1):
                 # All coming values in list should be larger than the current value.
                 if any(self.data[k] > self.data[key] for k in energy_keys[index:]):
-                    self.data = {}
-                    raise UpdateFailed("Invalid energy values, skipping update")
+                    LOGGER.info(
+                        "Ignoring invalid energy value %s for %s", self.data[key], key
+                    )
```

Comment:
Not sure if it's more `INFO` or `DEBUG`...

Reply:
Lets try INFO first and adjust later if needed.